dspace-api/src/main/java/sk/dtq/dspace/app/util/ACE.java

```diff
@@ -0,0 +1,196 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package sk.dtq.dspace.app.util;
+
+import java.util.Set;
+
+import org.apache.log4j.Logger;
+
+/**
+ * Class that represents single Access Control Entry
+ *
+ * @author Michal Josífko
+ * @author milanmajchrak
+ */
+
+public class ACE {
+
+    /** Logger */
+    private static final Logger log = Logger.getLogger(ACE.class);
+
+    private static final String POLICY_KEYWORD = "policy";
+
+    private static final String POLICY_DENY_KEYWORD = "deny";
+
+    private static final String POLICY_ALLOW_KEYWORD = "allow";
+
+    private static final String ACTION_KEYWORD = "action";
+
+    private static final String ACTION_READ_KEYWORD = "read";
+
+    private static final String ACTION_WRITE_KEYWORD = "write";
+
+    private static final String GRANTEE_TYPE_KEYWORD = "grantee-type";
+
+    private static final String GRANTEE_TYPE_USER_KEYWORD = "user";
+
+    private static final String GRANTEE_TYPE_GROUP_KEYWORD = "group";
+
+    private static final String GRANTEE_ID_KEYWORD = "grantee-id";
+
+    private static final String ANY_KEYWORD = "*";
+
+    public static final int ACTION_READ = 1;
+
+    public static final int ACTION_WRITE = 2;
+
+    private static final int POLICY_DENY = 1;
+
+    private static final int POLICY_ALLOW = 2;
+
+    private static final int GRANTEE_TYPE_USER = 1;
+
+    private static final int GRANTEE_TYPE_GROUP = 2;
+
+    private static final String GRANTEE_ID_ANY = "-1";
+
+    private int policy;
+
+    private int action;
+
+    private int granteeType;
+
+    private String granteeID;
+
+    /**
+     * Creates new ACE object from given String
+     *
+     * @param s
+     * @return ACE object or null
+     */
+
```

Comment:
is the newline according to our coding style?
does java(/dspace) has something like a codestyle linter?